### PR TITLE
Fixes #61: Add kubectl

### LINKED ISSUE #61: the pcp images are missing kubectl
During rhel7 conversion, the kubectl binary went missing from the pcp-central-logger pod.  This was formerly added via the centos origin-clients rpm.  No equivalent has been identified for a pure rhel7 image yet.  Could you hunt it down and get it into https://github.com/rhdt/EL-Dockerfiles/blob/master/base/pcp/Dockerfile ?

### PR DESCRIPTION
**Pull request by jmelis**
EDIT: remove original comment, it doesn't apply anymore

### CODE REVIEW

**Comment by jmelis** [comment]
@fche is this enough or do we need `oc` binary as well?

File: base/pcp/Dockerfile [hunk 1]
```diff
@@ -2,9 +2,11 @@ FROM registry.access.redhat.com/rhel7
 ENV LANG=en_US.utf8
 
 COPY pcp.repo /etc/yum.repos.d/pcp.repo
-RUN yum install -y pcp pcp-pmda-prometheus pcp-manager pcp-debuginfo \
+RUN yum-config-manager --enable rhel-7-server-extras-rpms && \
+    yum install -y pcp pcp-pmda-prometheus pcp-manager pcp-debuginfo \
                    pcp-export-pcp2zabbix pcp-system-tools pcp-webapi \
-                   pcp-webapps pcp-pmda-postgresql && yum clean all && \
+                   pcp-webapps pcp-pmda-postgresql kubernetes-client && \
```
Comment (aditya-konarde) [inline base/pcp/Dockerfile]: Try a `origin-clients` install instead of kubernetes-client here?

Comment (jmelis) [reply base/pcp/Dockerfile]: origin-clients is not available in RHEL

**Comment by jmelis** [comment]
Closing because it looks like this is not needed in the pcp base image

**Comment by jmelis** [comment]
This is a new version of the fix, @fche please ignore previous comments.

**Comment by jmelis** [comment]
@riuvshin can you please review?

**Comment by jmelis** [comment]
@riuvshin my concern is if we should pin it to a specific version of CentOS like it was, or use the /7/ endpoint.

Example:
http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/
vs
http://mirror.centos.org/centos/7.5.1804/paas/x86_64/openshift-origin/

I'd say /7/ as it's a go binary and there shouldn't be any problems in what 7.x version it was compiled

**Pull request merged**